site/docs/reference/Connectors/capture-connectors/mongodb.md

```diff
@@ -44,19 +90,25 @@ MongoDB source connector.
+| **`/address`**          | Address                                                            | Host and port of the database. Optionally can specify scheme for the URL such as mongodb+srv://host.                                                                                                                                                                                                            | string  | Required         |
+| **`/user`**             | User                                                               | Database user to connect as.                                                                                                                                                                                                                                                                                    | string  | Required         |
+| **`/password`**         | Password                                                           | Password for the specified database user.                                                                                                                                                                                                                                                                       | string  | Required         |
+| `/database`             | Database                                                           | Optional comma-separated list of the databases to discover. If not provided will discover all available databases in the instance.                                                                                                                                                                              | string  |                  |
+| `/batchAndChangeStream` | Capture Batch Collections in Addition to Change Stream Collections | Discover collections that can only be batch captured if the deployment supports change streams. Check this box to capture views and time series collections as well as change streams. All collections will be captured in batch mode if the server does not support change streams regardless of this setting. | boolean |                  |
+| `/pollSchedule`         | Default Batch Collection Polling Schedule                          | When and how often to poll batch collections. Accepts a Go duration string like '5m' or '6h' for frequency-based polling or a string like 'daily at 12:34Z' to poll at a specific time (specified in UTC) every day. Defaults to '24h' if unset                                                                 | string  |                  |
```

Comment:
I think the default `'24h'` should be placed in the "Required/Default" column instead of tucked inside "Description".

Reply:
It's not really a "schema default" in the sense that it would get auto-populated in the UI from a `default` annotation.  Probably a distinction without a difference thing here, but the default is actually an empty input which the connector will interpret as `24h`.

Reply:
I see, I've been interpreting any values in the "Required/Default" column not as a "schema default" but rather as a more general "if you don't provide a value, the connector will use X". As long as the value that's used by the connector if the user doesn't provide one is mentioned somewhere, that sounds good.